onboard/catkin_ws/src/cv/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cv)
+
+
+## Find catkin macros and libraries
+find_package(catkin REQUIRED COMPONENTS
+    message_generation
```

Comment:
delete message_generation, that is done in the custom_msgs package

Reply:
Sounds good, we just made that change.